src/api/ApiResponder.php

```diff
@@ -1047,7 +1047,12 @@ public function process_request($args) {
      * @return string
      */
     protected function verify_function_access($args) {
-        if (array_key_exists('type', $args)) {
+        if (!is_array($args)) {
+            $result = array(
+                'ok' => FALSE,
+                'message' => '$args must be an array',
```

Comment:
Let's not interpolate an invalid variable that an unauthenticated caller handed us, just to be on the safe side.  :>)

I think it'd be fine just to say "Arguments to BM API functions must be arrays"

Reply:
Oh, no interpolation with single quotes, so it would print the literal. However, I'll change the message anyway.